changelog.txt

```diff
@@ -2,6 +2,12 @@
 
 Version 1.9.11, 1 February 2016
 
+The Checker Framework distribution now contains annotated versions
```

Comment:
We missed the deadline for this release, right?  So shouldn't this text appear for the March 1 release, so that when we merge it after the release, it merges without a conflict?